src/main/java/core/file/xml/XMLMatchLineupParser.java

```diff
@@ -128,12 +128,11 @@ private static MatchLineupPlayer createPlayer(Element ele) {
 
 		// nur wenn Player existiert
 		if (spielerID > 0) {
-			tmp = (Element) ele.getElementsByTagName("FirstName").item(0);
-			name = tmp.getFirstChild().getNodeValue();
-			tmp = (Element) ele.getElementsByTagName("LastName").item(0);
-			if (tmp.getFirstChild() != null) { // there are players without a lastname
-				name = name + " " + tmp.getFirstChild().getNodeValue();
-			}
+			// First- and LastName can be empty
+			name = getStringValue((Element)ele.getElementsByTagName("FirstName").item(0));
+			var lastName = getStringValue((Element)ele.getElementsByTagName("LastName").item(0));
+			if ( name.length()>0 && lastName.length()>0) name = name + " ";
+			name = name + lastName;
```

Comment:
is there not an else block missing ?

Reply:
no, it isn't. 

maybe we should create the name with a StringBuilder?!  This could be better readable. is it the better way to do string concatination?